backend/src/routes/v1/saveforme/index.js

```diff
@@ -8,7 +8,15 @@ const { requireToken } = require('../../../middlewares')
 const router = Router()
 
 const schema = Joi.object({
-  email: Joi.string().required,
+  email: Joi.string().required(),
+  service: Joi.string(),
+  data: Joi.object({
+    id: Joi.string(),
+    link: Joi.string().required(),
+    title: Joi.string().required(),
```

Comment:
Vai dar ruim quando você for dar o get, recomendo tirar o required

Reply:
Então, como data não é required, ele já faz essa lógica, porque ai ele exige esses parametros caso exista data